mmv1/products/firebasedatabase/api.yaml

```diff
@@ -42,6 +42,7 @@ objects:
         name: 'region'
         description: |
           A reference to the region where the Firebase Realtime database resides.
+          Check all available regions https://firebase.google.com/docs/projects/locations#rtdb-locations
```

Comment:
nit: maybe make the link like markdown? `Check all [available regions](https://firebase.google.com/docs/projects/locations#rtdb-locations)`